include/utils/perf_log.h

```diff
@@ -279,9 +328,13 @@ class PerfLog
 
   /**
    * The actual log.
+   *
+   * An unsorted_map would work fine here and would be asymptotically
+   * faster, but in my tests for our log sizes there was no
+   * improvement.
```

Comment:
This was exactly what I was going to ask about, thanks for testing it already!

Reply:
I almost just unthinkingly switched to unsorted_map after I had the prototype working with map, but I then ran across that "gifs of how different sorting algorithms work" gallery on Reddit, which got me reading about weird little O(N^2) algorithms that are actually still useful for small N, which reminded me that hey, our N here isn't exactly huge...

It's not obviously *slower* either, but it does require us to define our own pair hash function (or factor out the crummy one in the TopologyMap code), so it's definitely less terse.